core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -537,6 +537,25 @@ struct HIPParallelLaunchKernelInvoker<DriverType, LaunchBounds,
     HIPInternal::constantMemReusable[hip_device].release(
         std::move(lock), hip_instance->m_stream);
   }
+
+  static void create_parallel_launch_graph_node(
+      DriverType const &driver, dim3 const &grid, dim3 const &block, int shmem,
+      HIPInternal const *hip_instance) {
+    // Just use global memory; coordinating through events to share constant
+    // memory with the non-graph interface is not really reasonable since
+    // events don't work with Graphs directly, and this would anyway require
+    // a much more complicated structure that finds previous nodes in the
+    // dependency structure of the graph and creates an implicit dependence
+    // based on the need for constant memory (which we would then have to
+    // somehow go and prove was not creating a dependency cycle, and I don't
+    // even know if there's an efficient way to do that, let alone in the
+    // structure we currenty have).
```

Comment:
```suggestion
    // based on the need for constant memory.
```
The text in parentheses don't add much and it's not clear who "I" is.

Reply:
Hey @masterleinad. I'm not sure who "I" is either 😄 

Actually, this long comment is copy pasted from the cuda case:
- https://github.com/kokkos/kokkos/blob/b33c89a21247ec0011a219571114cb33fbd1de55/core/src/Cuda/Kokkos_Cuda_KernelLaunch.hpp#L612-L630

I'm not sure what's best. If we modify the comment block as you are suggesting, we have the disadvantage that the cuda and hip comment blocks start to diverge. Should I modify both the cuda and the hip comment blocks as you are suggesting? Or just delete the entire comment block here?

Reply:
leave the comment its fine.